src/script/interpreter.cpp

```diff
@@ -1354,6 +1357,16 @@ static bool VerifyWitnessProgram(const CScriptWitness& witness, int witversion,
         return set_success(serror);
     }
 
+    // Policy limit for stack size and stack element size
```

Comment:
Might make sense to stick this inside the witversion==0 block if we want to tie both behaviors to specific versions or neither if we don't.